docs/dev_guide/deply_noobaa_on_minikube.md

```diff
@@ -0,0 +1,126 @@
+Change the working directory to the local repository (in this example the local repository is inside `SourceCode` directory).
+```bash
+ cd ./SourceCode/noobaa-core
+```
+We will use minikube to run the tests. It is recommended to build all images on the minikube docker daemon. Configure your docker client to use minikube's docker run:
+```bash
+eval $(minikube docker-env)
+```
+### 3) Before Building The Images (Noobaa-Operator Tab)
+Change the working directory to the local repository (in this example the local repository is inside `SourceCode` directory).
+```bash
+ cd ./SourceCode/noobaa-operator
+```
+In order to build the CLI and the operator image run the following:
+```bash
+. ./devenv.sh
+```
+_Note: the file `devenv.sh` contains the command `eval $(minikube docker-env)`. We run the command `eval $(minikube docker-env)` prior to an image build (whether from noobaa core repository or noobaa operator repository)._
+### 4) Build Operator Images (Noobaa-Operator Tab)
+```bash
+make all
+```
+This will build the following:
+* noobaa-operator image with tag `noobaa/noobaa-operator:<major.minor.patch>` (for example: `noobaa/noobaa-operator:5.13.0`). this tag is used by default when installing with the CLI.
+* noobaa CLI. The `devenv.sh` script is setting an alias `nb` to run the local build of the CLI.
+
+### 5) Build Core Images (Noobaa-Core Tab)
+Run the following to build noobaa core image with the desired tag to build the tester image:
+```bash
+make noobaa
+```
+Change the tag name  `noobaa:latest noobaa-core:<tag-name>`, for example: 
+```bash
+docker tag noobaa:latest noobaa-core:my-deploy
+```
+### 6) Deploy Noobaa (Noobaa-Operator Tab)
+```bash
+nb install --mini --noobaa-image='noobaa-core:my-deploy'
+```
+_Note: We have the alias to `nb` from the step 'Build Operator'._
+
+The installation should take 5-10 minutes.
+Once noobaa is installed please notice that the phase is Ready, you will see it in the CLI logs:
+
+✅ System Phase is "Ready".
+
+You can see something similar to this when getting the pods:
+```
+> kubectl get pods
+NAME                                               READY   STATUS    RESTARTS   AGE
+noobaa-core-0                                      1/1     Running   0          51m
+noobaa-db-pg-0                                     1/1     Running   0          51m
+noobaa-default-backing-store-noobaa-pod-a586c55b   1/1     Running   0          47m
+noobaa-endpoint-6cf5cccfc6-rmdrd                   1/1     Running   0          47m
+noobaa-operator-5c959d5564-qzgqb                   1/1     Running   0          51m
+```
+
+### 7) Wait For Default Backingstore to Be Ready (Noobaa-Operator Tab)
+We will use the default backingstore pod to run the tests, we need it to be in phase Ready, run:
+```bash
+kubectl wait --for=condition=available backingstore/noobaa-default-backing-store --timeout=6m
+```
+## Workarounds:
+### 1) MacOS:
+Change the `mini` resources inside file system.go (before building the operator image):
+
```

Comment:
This step is true only for M1 macs, and is not required for x86 Mac.
another point is that it is a hack and we should not instruct it.
We should remove this and fix it if needed in the code.

Reply:
I'll change it to M1.
I know that @danny tried it in a [PR](https://github.com/noobaa/noobaa-operator/pull/1075) and closed it, I also opened an [issue](https://github.com/noobaa/noobaa-operator/issues/1017) for a developer's flag. 
Until we find a solution, I think the documentation should reflect how we **actually** deploy Noobaa on Minikube.